csrc/fusion_profiler.cpp

```diff
@@ -669,7 +669,7 @@ SegmentProfiler& FusionProfiler::segment(size_t idx) {
   return get()->segments_.at(idx);
 }
 
-void FusionProfiler::start(bool cupti_disable) {
+/*static*/ void FusionProfiler::start(bool cupti_disable) {
```

Comment:
Did you mean to leave the commented static?

Reply:
Yes. I tend to do this extensively to remind readers this is a static method.